schema.graphql

```diff
@@ -242,7 +243,49 @@ enum FeeMovementType {
 
 type TreasuryBalance @entity {
   id: ID!
-  owner: Juror! # TODO: this should work for type Address
+  owner: Juror! # TODO: this should work for type Bytes
   token: ERC20!
   amount: BigInt!
 }
+
+type SubscriptionModule @entity {
+  id: ID!
+  court: CourtConfig!
+  currentPeriod: BigInt!
+  feeAmount: BigInt!
+  feeToken: Bytes!
+  periodDuration: BigInt!
+  prePaymentPeriods: BigInt!
+  resumePrePaidPeriods: BigInt!
+  latePaymentPenaltyPct: BigInt!
+  governorSharePct: BigInt!
+  periods: [SubscriptionPeriod!] @derivedFrom(field: "instance")
+  subscribers: [Subscriber!] @derivedFrom(field: "instance")
+}
+
+type SubscriptionPeriod @entity {
+  id: ID!
+  feeToken: Bytes!
+  feeAmount: BigInt!
+  balanceCheckpoint: BigInt!
+  totalActiveBalance: BigInt!
+  collectedFees: BigInt!
+  instance: SubscriptionModule!
+  jurorClaims: [JurorSubscriptionFee!] @derivedFrom(field: "period")
+}
+
+type Subscriber @entity {
+  id: ID!
+  subscribed: Boolean!
+  paused: Boolean!
+  lastPaymentPeriodId: BigInt!
+  previousDelayedPeriods: BigInt!
+  instance: SubscriptionModule!
+}
+
+type JurorSubscriptionFee @entity {
+  id: ID!
+  juror: Juror!
```

Comment:
Is it possible to also add an attribute in the `Juror`  entity with `[JurorSubscriptionFee]`? 
Because one limitation that the graph has is that we cannot do nested queries. So adding this attribute will enable us to get directly all subscriptions periods claimed by a Juror.

Reply:
Sure we can :)